server/rules-engine/helper.js

```diff
@@ -1,5 +1,32 @@
 const rulesEngine = require('../services/rulesEngineService')
 
+const actionRuleData = {
+  FG1: {
+    quantityRuleID: 1,
+    removeFeaturesRuleID: 2,
+    upperboundFact: 'adjustedPerimeter',
+    defaultUpperbound: 'totalPerimeter'
+  },
+  SW6: {
+    quantityRuleID: 7,
+    removeFeaturesRuleID: 6,
+    upperboundFact: 'pondlessArea',
+    defaultUpperbound: 'totalArea'
+  },
+  LV7: {
+    // LV7 has different eligibility criteria as only one rule needs to be true
+    // so we override the default eligibility check with this function
+    // Logic is also a bit odd: if there are no eligibility rules enabled then
+    // we assume you are eligible, otherwise either of one of the two eligibilty rules
+    // must pass if enabled. It's not ideal but works for this limited use case.
+    eligibilityCheckOverride: (result, enabledRules) => {
+      const ruleNames = enabledRules.map((rule) => rule.event.type)
+      const passedRuleNames = result.events.map((event) => event.type)
+      return ruleNames.length > 0 ? ruleNames.some((rule) => passedRuleNames.includes(rule)) : true
```

Comment:
If the water pollution, reintroduced grazing and SSSI rules are all enabled, this check return true for a parcel which **only** passes the SSSI rule. That's not correct as the requirement should be that one or both of the water pollution or grazing rules pass **in addition to** the SSSI rule.

Reply:
Agreed,  having thought about it for I while I realised there was no nice way of doing this properly in a sensible amount of time so went with a non-ideal solution for the reasons you previously sate about this being a time limited prototype. But yes it should be fixed going forward if this stuff is going to be reused.